### PR TITLE
Compare results with alexa

### LINKED ISSUE #20: Compare results with Alexa

### PR DESCRIPTION
**Pull request by tiriplicamihai**
Fixes #20

### CODE REVIEW

**Comment by tiriplicamihai** [comment]
cc @piatra can you please review? We couldn't use the Alexa spider because scrapy is async.

File: src/data_set_builder/bag_of_words.py [hunk 1]
```diff
@@ -104,23 +130,32 @@ def main():
...
-            prediction[1] = prediction[1] + abs(x[2]) # female_confidence is neg
+            prediction[1] += abs(x[2]) # female_confidence is neg
+            alexa_result[0] += x[4]
 
     # normalize and print the prediction
+    print 'Prediction:'
     print [float(x) / sum(prediction) for x in prediction]
 
+    print 'Alexa:'
+    print [float(x) / sum(alexa_result) for x in alexa_result]
```
Comment (piatra) [inline src/data_set_builder/bag_of_words.py:158]: @tiriplicamihai Let's try and use http://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html for computing accuracy. This is based on feedback received during the last project presentation.

Comment (piatra) [inline src/data_set_builder/bag_of_words.py:158]: Preferably we would want to plot everything for the presentation http://scikit-learn.org/stable/auto_examples/model_selection/plot_roc.html
